lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -352,18 +297,20 @@ sub _check_limits {
     my $limit = $self->limit;
     my $log   = $self->log;
 
-    eval {
-        my $db      = $self->sqlite->db;
-        my $results = $db->select('assets', [qw(filename size last_use)], undef, {-asc => 'last_use'});
-        for my $asset ($results->hashes->each) {
-            my $asset_size = $asset->{size} || -s $asset->{filename};
-            $self->_decrease($asset_size) if $self->purge_asset($asset->{filename});
-            last if !$self->_exceeds_limit($needed);
-        }
-    } if $self->_exceeds_limit($needed);
-    if ($@) { $log->error("CACHE: check_limit failed: $@") }
-
-    $log->debug("CACHE: Health: Real size: $self->{cache_real_size}, Configured limit: $limit");
+    if ($self->_exceeds_limit($needed)) {
+        $log->info("Cache size $self->{cache_real_size} + $needed exceeds limit of $limit, purging least used assets");
```

Comment:
In the output it would not be clear what is the meaning of the individual values, e.g. `"Cache size 1234 + 123 exceeds limit of 1240, purging least used assets"`. I suggest to prefix the values with its meaning, e.g.

```perl
$log->info("Cache size '$self->{cache_real_size}' + needed '$needed' exceeds limit of '$limit', purging least used assets");
```

and please add a unit. At best we can use a perl module to output human readable sizes.

Reply:
I consider human readable sizes out of scope for this PR, but it can certainly be part of a followup.

Reply:
agreed but the notice regarding the explanation of the value still holds

Reply:
Btw, we have https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Utils.pm#L753 so it should be easy